### PR TITLE
Examples in our language ☭

### LINKED ISSUE #25: Find and write game structures in LCGS
We need multiple LCGS programs for testing later. We can probably find some in the ATL paper or PRISM-games.

### PR DESCRIPTION
**Pull request by GeneralDenmark**
Fixes #25 

Examples which compile in LCGS and at least one working formula:
- [ ] Mexican standoff

### CODE REVIEW

**Comment by jenrik** [comment]
If we formulate a couple of properties/formulas for each example this would be ideal for use as tests and benchmarks

**Comment by jenrik** [comment]
[Skriv properties/formulas til vores LCGS eksempler](https://trello.com/c/JMQnTKaR/69-skriv-properties-formulas-til-vores-lcgs-eksempler)

File: docs/lcgs/examples/Gossipping_Girls_Circular/Gossipping_Girls.lcgs [hunk 1]
```diff
@@ -13,54 +13,51 @@ player p4 = girl [friend1 = p3, friend2 = p1,
...
-			// Max received secret
-			max(
-				// Call left
-				((call_friend1 * friend1.call_friend2) * friend1.secret1), 
-				// Call right
-				((call_friend2 * friend2.call_friend1) * friend2.secret1)), 
+			// Call left
+			((call_friend1 * friend1.call_friend2) * friend1.secret1), 
+			// Call right
+			((call_friend2 * friend2.call_friend1) * friend2.secret1)),
```
Comment (GeneralDenmark) [inline docs/lcgs/examples/Gossipping_Girls_Circular/Gossipping_Girls.lcgs:23]: Du glemte at fjerne en parantes i slutningen af linjen

File: docs/lcgs/examples/Gossipping_Girls_Circular/Gossipping_Girls.lcgs [hunk 2]
```diff
@@ -13,54 +13,51 @@ player p4 = girl [friend1 = p3, friend2 = p1,
...
-			// Max received secret
-			max(
-				// Call left
-				((call_friend1 * friend1.call_friend2) * friend1.secret2), 
-				// Call right
-				((call_friend2 * friend2.call_friend1) * friend2.secret2)), 
+			// Call left
+			((call_friend1 * friend1.call_friend2) * friend1.secret2), 
+			// Call right
+			((call_friend2 * friend2.call_friend1) * friend2.secret2)),
```
Comment (GeneralDenmark) [inline docs/lcgs/examples/Gossipping_Girls_Circular/Gossipping_Girls.lcgs:34]: Du glemte at fjerne en parantes i slutningen af linjen

File: docs/lcgs/examples/Gossipping_Girls_Circular/Gossipping_Girls.lcgs [hunk 3]
```diff
@@ -13,54 +13,51 @@ player p4 = girl [friend1 = p3, friend2 = p1,
...
-			// Max received secret
-			max(
-				// Call left
-				((call_friend1 * friend1.call_friend2) * friend1.secret3), 
-				// Call right
-				((call_friend2 * friend2.call_friend1) * friend2.secret3)), 
+			// Call left
+			((call_friend1 * friend1.call_friend2) * friend1.secret3), 
+			// Call right
+			((call_friend2 * friend2.call_friend1) * friend2.secret3)),
```
Comment (GeneralDenmark) [inline docs/lcgs/examples/Gossipping_Girls_Circular/Gossipping_Girls.lcgs:45]: Du glemte at fjerne en parantes i slutningen af linjen

**Review by GeneralDenmark — COMMENTED** [review]
@cogitantium

File: docs/lcgs/examples/Gossipping_Girls_Circular/Gossipping_Girls.lcgs (hunk 1 above)
Comment (cogitantium) [reply docs/lcgs/examples/Gossipping_Girls_Circular/Gossipping_Girls.lcgs]: Fix i 1776c10

File: docs/lcgs/examples/Gossipping_Girls_Circular/Gossipping_Girls.lcgs (hunk 2 above)
Comment (cogitantium) [reply docs/lcgs/examples/Gossipping_Girls_Circular/Gossipping_Girls.lcgs]: Fix i 1776c10

File: docs/lcgs/examples/Gossipping_Girls_Circular/Gossipping_Girls.lcgs (hunk 3 above)
Comment (cogitantium) [reply docs/lcgs/examples/Gossipping_Girls_Circular/Gossipping_Girls.lcgs]: Fix i 1776c10

**Comment by jenrik** [comment]
Some formula files have multiple formulas in them. These files need to be split into one formula per file, otherwise we can't load them

**Review by cogitantium — APPROVED** [review]

**Review by NicEastvillage — APPROVED** [review]

**Review by jenrik — APPROVED** [review]
YOLO merge it!

**Pull request merged**